user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+---
+uid: DOM_Remove_Enum_Entry
+---
+
+# How to remove an enum entry from a GenericEnumFieldDescriptor
+
+This tutorial shows you how to soft-delete and permanently delete an enum entry from a GenericEnumFieldDescriptor.
+
+## Overview
+
+- [Step 1: Setup DOM](#step-1-setup-dom)
+- [Step 2: Soft-delete the enum entry](#step-2-soft-delete-the-enum-entry)
+- [Step 3: Remove the value from existing instances](#step-3-remove-the-value-from-existing-instances)
+- [Step 4: Remove the enum entry from the DOM definition](#step-4-remove-the-enum-entry-from-the-dom-definition)
+
+The soft-delete option on an enum entry hides that entry to users. That way users are unable to create more DOM instances using that entry. Later on the entry can be completely removed.
+If the entry is used by a DOM instance, that value needs to be removed from those instances before removing the entry from the definition. If that enum entry is not used, it can immediately removed (see [Step 4: Remove the enum entry from the DOM definition](#step-4-remove-the-enum-entry-from-the-dom-definition)).
```

Comment:
```suggestion
If the entry is used by a DOM instance, that value needs to be removed from those instances before removing the entry from the definition. If that enum entry is not used, it can immediately be removed (see [Step 4: Remove the enum entry from the DOM definition](#step-4-remove-the-enum-entry-from-the-dom-definition)).
```